rosidl_default_runtime/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,109 @@
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`rosidl_default_runtime` does not currently have its own features.
+
+### Public API Documentation [3.ii]
+
+`rosidl_default_runtime` does not have a public API and therefore does not require API documentation.
+
+### License [3.iii]
+
+The license for `rosidl_default_runtime` is Apache 2.0, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the [LICENSE](../LICENSE) file.
+
+There are no source files in this package and therefore linters do not check for a license statement.
+
+### Copyright Statements [3.iv]
+
+There are no source code files in this package and therefore no files are currently yet copyrighted.
+
+## Testing [4]
+
+`rosidl_default_runtime` is a package providing solely CMake files and therefore does not require tests and has no coverage or performance requirements.
+
+### Linters and Static Analysis [4.v]
+
+`rmw` uses and passes all the standard linters and static analysis tools for a CMake package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters).
+
+Results of linter tests can be found [here](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rosidl_default_runtime/).
+
+## Dependencies [5]
+
+### Direct Runtime ROS Dependencies [5.i]/[5.ii]
+
+`rosidl_default_runtime` has the following group dependencies:
+* `rosidl_runtime_packages`
+* `rosidl_typesupport_c_packages`
+* `rosidl_typesupport_cpp_packages`
+
+### Direct Runtime Non-ROS Dependencies [5.iii]
+`rosidl_default_runtime` does not have any runtime non-ROS dependencies.
+
+## Platform Support [6]
+
+`rosidl_default_runtime` supports all of the tier 1 platforms as described in [REP-2000](https://www.ros.org/reps/rep-2000.html#support-tiers), and tests each change against all of them.
+
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rosidl_default_runtime/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rosidl_default_runtime/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rosidl_default_runtime/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rosidl_default_runtime/)
+
+## Vulnerability Disclosure Policy [7.i]
+
+This package does not yet have a Vulnerability Disclosure Policy
```

Comment:
Consider adding a table at the end summarizing the current state of the package

Reply:
I guess I tend to think the issue (https://github.com/ros2/rosidl_defaults/issues/6) makes it pretty clear the current state of the package and what else needs to be done, and the table just adds a third place where the current state of the package needs to be tracked and updated.